object-store-client-play-26/src/main/scala/uk/gov/hmrc/objectstore/client/play/PlayWSHttpClient.scala

```diff
@@ -84,7 +88,7 @@ class PlayWSHttpClient @Inject()(wsClient: WSClient, config: ObjectStoreClientCo
     logger.info(s"Request: Url: $url")
     body.flatMap { httpBody =>
       val hdrs = (headers ++ httpBody.length.map("Content-Length" -> _.toString) ++ httpBody.md5.map(
-        "Content-MD5" -> _)) :+ authorizationHeader
+        "Content-MD5" -> _))
```

Comment:
I wonder if these headers should also move?
If this class is a generic http client, are they always required?